feature/visited/src/main/kotlin/uk/govuk/app/visited/data/VisitedItemsTransformer.kt

```diff
@@ -0,0 +1,78 @@
+package uk.govuk.app.visited.data
+
+import uk.govuk.app.visited.data.model.VisitedItem
+import uk.govuk.app.visited.ui.model.VisitedUi
+import java.time.LocalDate
+
+class VisitedItemsTransformer(
+    private val visitedItems: List<VisitedItem>,
+    private val todaysDate: LocalDate
+) {
+    val todaysItems: List<VisitedItem>
+    val thisMonthsItems: List<VisitedItem>
+    val previousMonthsItems: List<VisitedItem>
+    val groupedPreviousMonthsItems: Map<String, List<VisitedItem>>
+
+    init {
+        todaysItems = todaysItems()
+        thisMonthsItems = thisMonthsItems()
+        previousMonthsItems = previousMonthsItems()
+        groupedPreviousMonthsItems = groupPreviousMonthsItems()
+    }
+
+    fun transform(): Map<String, List<VisitedUi>> {
+        var groupedVisitedItems = emptyMap<String, List<VisitedUi>>()
+
+        if (todaysItems.isNotEmpty()) {
+            groupedVisitedItems += mapOf("Today" to toVisitedUi(todaysItems))
```

Comment:
Are the keys visible on the uI?

Reply:
Yes, so prob should be string values - will fix

Reply:
Fixed up

Reply:
Solution is not great, but at least there's only one place to update now.

Reply:
Hard coded strings still appear to be present

Reply:
That's odd 🤔 They should have been replaced via this commit: https://github.com/alphagov/govuk-mobile-android-app/pull/91/commits/1767724e4aa865643fea8d089635bc933be4c104

Reply:
Admittedly, there are still strings in the SectionTitles file

Reply:
I'll try to replace them with @StringRef